drupal-ukraine/dmaps
# #14: Location: Convert hook_permission()

Issue #14 

Steps for review:
1. Login as admin
2. Go to "/admin/people/permissions"
3. Make sure you see following permissions for dmaps:
- View location directory
- View node location table
- View user location table
- Submit latitude/longitude
